File: more_exercises/02_conditional_statements/02_sleepy_tom_cat.py
days_off = int(input())
time_days_off = days_off * 127
time_working_days = (365 - days_off) * 63
playing_time = time_days_off + time_working_days
norm = 30000
difference = abs(norm - playing_time)
hours = difference // 60
minutes = difference % 60

if playing_time > norm:
    print("Tom will run away")
    print(f"{hours} hours and {minutes} minutes more for play")
elif playing_time <= norm:
    print("Tom sleeps well")
    print(f"{hours} hours and {minutes} minutes less for play")

File: 05_JavaScriptBasics/03_MoreExercises/02_ConditionalStatements/02_sleepyTomCat.js
function sleepyCatTom(input){
    let norm = 30000
    let yearDays = 365
    let workingDayQuota = 63
    let holidaysQuota = 127

    let holidays = Number(input[0])
    let holidayMinutes = holidays * holidaysQuota
    let workingDaysMinutes = (yearDays - holidays) * workingDayQuota
    let totalMinutes = holidayMinutes + workingDaysMinutes

    let difference = Math.abs(totalMinutes - norm)
    let hours = Math.trunc(difference / 60)
    let minutes = difference % 60

    if (totalMinutes > norm){
        console.log("Tom will run away")
        console.log(`${hours} hours and ${minutes} minutes more for play`)
    } else {
        console.log("Tom sleeps well")
        console.log(`${hours} hours and ${minutes} minutes less for play`)
    }
}

sleepyCatTom(["20"])
sleepyCatTom(["113"])
